packages/castmill/lib/castmill/workers/image_transcoder.ex

```diff
@@ -52,7 +52,7 @@ defmodule Castmill.Workers.ImageTranscoder do
           {uri, size} =
             Image.open!(stream)
             |> Image.thumbnail!(size)
-            |> upload_image(organization_id, media_id, "#{context}.#{extension}")
+            |> upload_image(organization_id, media_id, "#{context}#{extension}")
```

Comment:
Is this correct? we need the period to get a file extension, like "preview.jpg", otherwise we would get "previewjpg"

Reply:
extension already includes the `"."` I noticed that we had files like "preview..jpg" and changed this to get rid of the double dots.